modules.d/50plymouth/module-setup.sh

```diff
@@ -1,27 +1,77 @@
 #!/bin/bash
 
 pkglib_dir() {
-    local _dirs="/usr/lib/plymouth /usr/libexec/plymouth/"
+    local _dirs="/usr/lib/plymouth /usr/lib64/plymouth /usr/libexec/plymouth"
     if find_binary dpkg-architecture &> /dev/null; then
```

Comment:
This is an independent issue that I see during a quick review: `dpkg-architecture` is shipped by dpkg-dev and is not expected to be present on all systems. The arch directory should be derived via a different approach.

Reply:
`find_binary dpkg-architecture` succeeds only if the `dpkg-architecture` command is found.